src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -41,6 +41,12 @@
      */
     public String phoneNumber;
 
+    /**
+     *  The user's preferred language, in language tag format
+     *  e.g. 'en-US', 'fr-FR', 'es-ES', 'zh-CN', etc.
+     * **/
```

Comment:
Remove extra star and space.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/164/commits/1d06681bccdd0db9bfcd7de8d02454b7f0d5c15c